src/Superstructure/ESMFMod/interface/ESMF_Init_C.F90

```diff
@@ -59,14 +59,16 @@ subroutine f_esmf_frameworkinitialize(lang, configFileName, &
 
    end subroutine f_esmf_frameworkinitialize
 
-   subroutine f_esmf_frameworkfinalize(rc)
+   subroutine f_esmf_frameworkfinalize(rc, endFlag)
        use ESMF_CompMod
        use ESMF_InitMod
+       use ESMF_UtilTypesMod
 
        implicit none
 
        integer :: rc
+       type(ESMF_End_Flag) :: endFlag
 
-       call ESMF_Finalize(rc=rc)
+       call ESMF_Finalize(endflag=endFlag, rc=rc)
```

Comment:
```suggestion
       call ESMF_Finalize(rc=rc, endflag=endFlag)
```
Tiny nitpick, just to keep the arguments in the same order.

Reply:
I think you might've missed the commit message just before 😅 -- putting `rc` at the end is an ESMF style convention